andela/baby-face-cfh
# #153742471 Card czar draws card on game start

#### What does this PR do?
Implements feature allowing card czar to draw card on the start of a new game round.

#### Description of Task to be completed?
As a CZAR I should be prompted to click to select from the deck of black cards the next question card.

#### How should this be manually tested?
- Visit [Cards for humanity - baby-face](http://baby-face-cfh-staging.herokuapp.com/#!/app)
- Duplicate tabs to get up to three new players for game to commence
- Start a new game and observe as card czar is prompted to choose from the listed cards

#### What are the relevant pivotal tracker stories?
#153742471

#### Screenshots (if appropriate)

<img width="1440" alt="screen shot 2018-01-14 at 14 01 05" src="https://user-images.githubusercontent.com/30455641/34919020-7b2a7770-f95d-11e7-9ce4-a4c4985ef9e8.png">
<img width="1440" alt="screen shot 2018-01-14 at 14 01 21" src="https://user-images.githubusercontent.com/30455641/34919021-7b64103e-f95d-11e7-854f-76ca1ac2a021.png">
